developer/ruby/GitHubIssueStats.rb

```diff
@@ -0,0 +1,148 @@
+require 'github_api'
+require 'date'
+require 'yaml'
+
+# Get the begin date, by finding the last major/minor (X.Y.0) release
+# that is at least a week old
+def get_begin_date_and_previous_tag()
+  a_week_ago = Time.now - (60*60*24*7)
+
+  @github.repos.releases.list(owner: @repo_owner,
+                              repo: @repo).each_page do |page|
+    page.each do |release|
+      next if release.tag_name !~ /^v\d\.\d\.0$/
+      release_date = Time.parse(release.created_at)
+      next if release_date > a_week_ago
+      puts "Found previous major/minor release: #{release.tag_name}, #{release_date}"
```

Comment:
Maybe skip pre-releases?

Reply:
`next if release.prerelease`

Reply:
Matching to `/^v\d\.\d\.0$/`, so that will skip anything that isn't tagged like X.Y.Z.  A pre-release should have a a suffix, like -alpha1 or -rc1 or whatnot.